packages/safe-core-sdk/README.md

```diff
@@ -68,13 +68,36 @@ const safeFactory = await SafeFactory.create({ ethAdapter })
 
 const owners = ['0x<address>', '0x<address>', '0x<address>']
 const threshold = 3
-const safeAccountConfig: SafeAccountConfig = { owners, threshold }
+const safeAccountConfig: SafeAccountConfig = {
+  owners,
+  threshold,
+  // ...
+}
 
 const safeSdk: Safe = await safeFactory.deploySafe(safeAccountConfig)
 ```
 
 The method `deploySafe` executes a transaction from `owner1` account, deploys a new Safe and returns an instance of the Safe Core SDK connected to the new Safe.
 
+The `SafeFactory` will deploy by default the last version of the Safe contracts available (currently `v1.3.0`). To deploy an older version of the Safe contracts instantiate the `SafeFactory` adding the property `safeVersion` with the desired version number.
```

Comment:
"The `SafeFactory` will deploy the last version of the Safe contracts available by default"